include/pokemon_jump.h

```diff
@@ -0,0 +1,14 @@
+#ifndef GUARD_POKEMON_JUMP_H
+#define GUARD_POKEMON_JUMP_H
+
+#include "main.h"
+
+void sub_802A9A8(u16 monId, MainCallback callback);
+bool32 sub_802C908(u16 species);
+void sub_802C920(void);
+void ResetPokeJumpResults(void);
```

Comment:
I think only this declaration makes sense in this file. Those `sub_*` are not even functions in firered.

Reply:
Yeah, quite a few .h files have pokeemerald function names in them so we should look at cleaning them up eventually.

Reply:
Yes, most should be lazy copies at an early stage. but if we know they're incorrect, there's no reason to include them.